lib/cartopy/mpl/geoaxes.py

```diff
@@ -1364,7 +1364,7 @@ def pcolor(self, *args, **kwargs):
 
     def quiver(self, x, y, u, v, *args, **kwargs):
         """
-        Plot a 2-D field of arrows.
+        Plot a field of arrows.
```

Comment:
In the rare case of those who read the docs someone might think that [this](http://nbviewer.ipython.org/gist/ocefpaf/40e024cbd568d5353a49) is not possible.